packages/design-system/src/components/UsaBanner/UsaBanner.tsx

```diff
@@ -1,8 +1,16 @@
-import React, { useState } from 'react';
-import { Button } from '../Button';
+/* eslint-disable react/no-multi-comp */
+/* eslint-disable react/prop-types */
```

Comment:
ES lint was complaining about these errors. Specifically the `prop-types` error is confusing to me because it was complaining about `locale` not being in prop validation when it is included in the interface.

I wonder if it's because `locale` is an optional prop and maybe needs a default?

Reply:
I went down a rabbit hole thinking that maybe the fact that our version of the eslint TypeScript plugin doesn't support our newer version of TypeScript has something to do with it. Unfortunately we can't update that plugin/parser without updating all of eslint, it seems. This error is certainly puzzling otherwise 🤔